.dotnet/tests/Samples/Chat/Sample01_SimpleChat.cs

```diff
@@ -12,10 +12,10 @@ public void Sample01_SimpleChat()
         {
             ChatClient client = new("gpt-3.5-turbo", Environment.GetEnvironmentVariable("OPENAI_API_KEY"));
 
-            ChatCompletion chatCompletion = client.CompleteChat("How does AI work? Explain it in simple terms.");
+            ChatCompletion chatCompletion = client.CompleteChat([new UserChatMessage("How does AI work? Explain it in simple terms.")]);
 
             Console.WriteLine($"[ASSISTANT]:");
-            Console.WriteLine($"{chatCompletion.Content}");
+            Console.WriteLine($"{chatCompletion.Content[0].Text}");
```

Comment:
Also not for this PR -- I wonder if could use our own custom type (`MessageContentPartList : IReadOnlyList<MessageContentPart>`) such that we could override `ToString()` to be friendly to happy-path `Console.WriteLine(chatCompletion.Content);`

Reply:
I like this idea!